src/platform/site-wide/sass/modules/_m-homepage.scss

```diff
@@ -212,7 +212,7 @@
     &.fa-user-circle {
       font-size: 1.5em;
       padding-right: .3em;
-      
+
```

Comment:
```suggestion
```

Reply:
good catch. I didn't even notice this as my editor is the one who removed the trailing whitespace.